src/app/page.tsx

```diff
@@ -1,23 +1,47 @@
-import { PetGrid } from '@/components/PetGrid';
-import { AddPetModal } from '@/components/modals/AddPetModal';
-import { Toaster } from '@/components/ui/toaster';
 import { font_accent } from '@/components/ui/fonts';
 import Link from 'next/link';
 import { Button } from '@/components/ui/button';
 import '@/app/styles/main.scss';
 
 export default function Home() {
   return (
-    <div className="grid place-items-center h-[100vh] text-center doors-of-durin">
+    <div className="h-[100vh] w-full home">
       <main className="text-center">
-        <h1 className={`${font_accent.className} font-semibold text-3xl`}>
-          Pet app home page
-        </h1>
-        <div>
-          <p>Speak, friend, and</p>
-          <Link href="user">
-            <Button>Enter</Button>
-          </Link>
+        <div className="home-img home-content">
+          <div className="home-img__blob home-img__blob--first"></div>
+          <div className="home-img__blob home-img__blob--second"></div>
+          <div className="home-img__blob home-img__blob--third"></div>
+          <div className="home-img__blob home-img__blob--fourth"></div>
```

Comment:
Why did you choose BEM? I suppose giving them a class of `.blob` and a `nth-of-type` selector would suffice :)